pylint_django/transforms/foreignkey.py

```diff
@@ -87,10 +100,24 @@ def infer_key_classes(node, context=None):
                 module_name = current_module.name
             elif not module_name.endswith('models'):
                 # otherwise Django allows specifying an app name first, e.g.
-                # ForeignKey('auth.User') so we try to convert that to
-                # 'auth.models', 'User' which works nicely with the `endswith()`
-                # comparison below
-                module_name += '.models'
+                # ForeignKey('auth.User')
+
+                django_spec = find_spec('django')
+                use_django_model_resolution = bool(django_spec)
+
+                if use_django_model_resolution:
+                    # If Django is installed we can use it to resolve the module name
+                    try:
+                        module_name = _module_name_from_django_model_resolution(model_name, module_name)
+                    except LookupError:
+                        use_django_model_resolution = False
+
+                if not use_django_model_resolution:
```

Comment:
that's an `else` to the above block, isn't it ?

Reply:
Yes, but I didn't use an `else` because I wanted to run this logic if django's resolution failed. The value of `use_django_model_resolution` is updated on line 113, inside of the `if` block above.

Reply:
No longer checking to see if Django is installed so these specific ifs aren't needed anymore. Resolving conversation.